configs/fourdag/README.md

```diff
@@ -0,0 +1,112 @@
+# FourDAG
```

Comment:
Using the full name for the title would be good. Besides, we can add a short explanation for the abbr.

For example,

```
# 4D Association Graph for Realtime Multi-person Motion Capture Using Multiple Video Cameras

As a python variable name cannot start with a number, we refer to this method as `FourDAG` in the following text and code.
```